simulatedDevice.py

```diff
@@ -0,0 +1,185 @@
+import asyncio
+import random
+import RPi.GPIO as GPIO
+import datetime
+import time
+import uuid
+from azure.iot.device.aio import IoTHubDeviceClient
+from azure.iot.device import Message
+
+movement_counter = 0
+doorstate = "closed"
+roomstate = "vacant"
+timeMovementCounterStarted = datetime.datetime.now()
```

Comment:
You should be able to import datetime this way from **datetime import datetime** and then you can just **datetime.now()**